avaframe/com1DFA/com1DFA.py

```diff
@@ -211,8 +212,9 @@ def com1DFACoreTask(simDict, inputSimFiles, avalancheDir, outDir, cuSim):
     # load configuration object for current sim
     cfg = simDict[cuSim]["cfgSim"]
 
-    # check configuraton for consistency
+    # check configuraton for consistency and plausibility
     checkCfg.checkCfgConsistency(cfg)
+    checkCfg.checkCfgPlausibility(cfg)
```

Comment:
See my comment at the function itself. If naming ist the problem: just use ceckCfg ...